src/network/networkctl-misc.c

```diff
@@ -18,66 +20,41 @@
 #include "polkit-agent.h"
 #include "set.h"
 #include "string-util.h"
+#include "strv.h"
+#include "strv-fundamental.h"
```

Comment:
Please do not directly include `xyz-fundamental.h`.
`strv-fundamental.h` is included by `strv.h`. Hence, please drop it.